com.toggl.TogglDesktop.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id": "com.toggl.TogglDesktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "rename-desktop-file": "toggldesktop.desktop",
+    "rename-appdata-file": "toggldesktop.appdata.xml",
+    "rename-icon": "toggldesktop",
+    "command": "TogglDesktop.sh",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host",
+        "--share=network",
+        "--share=ipc",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "TogglDesktop",
+            "buildsystem": "cmake",
```

Comment:
does this work with cmake-ninja instead? If so it'll be much faster to build.

Reply:
it does, sorry I didn't put it there in the first place, I wasn't following the development around flatpak too closely in the past so I didn't know this option was in place
i just switched it to ninja